docker/run-mysql.sh

```diff
@@ -7,6 +7,10 @@ start() {
     echo "attempting to remove old $CONTAINER container..."
     docker rm -f $CONTAINER || echo "docker rm failed. $CONTAINER not found."
 
+    # this is a necessary step for m1 laptops since a linux/arm64 image does not exist for mysql:5.6
+    echo "pulling mysql docker image"
+    docker pull --platform linux/x86_64 mysql:5.6
```

Comment:
would this break for non m1 laptop cuz it'll always download this image and use it for non m1 laptop too?

Reply:
this command pulls the non-m1 image, so it will not affect non-m1 laptops. (There is no m1 / arm64 image which was causing issues on m1 laptops. So that's why this command pulls the x86_64 image.)

Also, if the image already exists then the image is not downloaded again:
```
pulling mysql docker image
5.6: Pulling from library/mysql
Digest: sha256:20575ecebe6216036d25dab5903808211f1e9ba63dc7825ac20cb975e34cfcae
Status: Image is up to date for mysql:5.6
docker.io/library/mysql:5.6
starting up mysql container...
```

Reply:
so this image is for non-m1 laptop? how does this fix things for m1 laptop then?

also I tagged u in https://github.com/DataBiosphere/leonardo/pull/2983#issuecomment-1319119036, which I was hoping for some comments :)

Reply:
I think m1 laptops can run x86_64 images using an emulator (Rosetta) but not the other way around.

Reply:
so for intel machines it doesn't make a difference but for m1 laptops, it provides an image that 1) exists, and 2) can be run on m1 machines.

I thumbed #2983 because that seems to be a more elegant solution than this one